.github/workflows/slack-develop-commit.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  push:
+    branches:
+      - 'develop'
+
+jobs:
+  new_push_job:
+    runs-on: ubuntu-latest
+    name: New push to repo
+    steps:
+    - name: Send GitHub trigger payload to Slack Workflow Builder
+      id: slack
+      uses: slackapi/slack-github-action@v1.23.0
+      with:
+        payload: |
+          {
+            "text": "New commit merged to develop. @cc-pwa-kit-team please review and reconcile",
+            "blocks": [
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "<a target="_blank" href="https://github.com/SalesforceCommerceCloud/pwa-kit/commit/${{ github.sha }}">View Commit</a>"
+                }
+              }
+            ]
```

Comment:
I agree with Adam that this should work! Have you tried testing this in another channel?

Another thing that may be helpful to this notification is **who** completed this merge, so feature branch owners know who to contact right away. Seems like we can pass this info to Slack with: `${{ github.actor }}`